deploy/helm/kubecf/assets/operations/instance_groups/api.yaml

```diff
@@ -177,44 +69,7 @@
   value: "/var/vcap/packages/"
 
 # Enable volume services
-- type: replace
-  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/volume_services_enabled?
-  value: true
-
-# Add empty BPM processes to buildpacks.
-- type: replace
-  path: /instance_groups/name=api/jobs/name=binary-buildpack/properties?/quarks/bpm/processes
-  value: []
-- type: replace
-  path: /instance_groups/name=api/jobs/name=dotnet-core-buildpack/properties?/quarks/bpm/processes
-  value: []
-- type: replace
-  path: /instance_groups/name=api/jobs/name=go-buildpack/properties?/quarks/bpm/processes
-  value: []
-- type: replace
-  path: /instance_groups/name=api/jobs/name=java-buildpack/properties?/quarks/bpm/processes
-  value: []
-- type: replace
-  path: /instance_groups/name=api/jobs/name=nodejs-buildpack/properties?/quarks/bpm/processes
-  value: []
-- type: replace
-  path: /instance_groups/name=api/jobs/name=nginx-buildpack/properties?/quarks/bpm/processes
-  value: []
-- type: replace
-  path: /instance_groups/name=api/jobs/name=r-buildpack/properties?/quarks/bpm/processes
-  value: []
-- type: replace
-  path: /instance_groups/name=api/jobs/name=php-buildpack/properties?/quarks/bpm/processes
-  value: []
-- type: replace
-  path: /instance_groups/name=api/jobs/name=python-buildpack/properties?/quarks/bpm/processes
-  value: []
-- type: replace
-  path: /instance_groups/name=api/jobs/name=ruby-buildpack/properties?/quarks/bpm/processes
-  value: []
-- type: replace
-  path: /instance_groups/name=api/jobs/name=staticfile-buildpack/properties?/quarks/bpm/processes
-  value: []
+{{- include "_capi.setProperty" (list "volume_services_enabled" true) }}
```

Comment:
If I understand `git blame` correctly, this is used for eirini-persi?  That is, I'm not sure there's anything we can use here for diego (since nfs-volume is definitely not working).  Probably not worse conditioning this on eirini though?

Reply:
I have no idea; I just took what already existed and moved it to the new mechanism that sets the property in all related CAPI jobs.

I think changing the logic here is unrelated to the current PR and I don't want to risk introducing a regression.